src/main/java/de/tum/cit/aet/artemis/atlas/domain/competency/LearningPath.java

```diff
@@ -31,6 +31,12 @@ public class LearningPath extends DomainObject {
     @Column(name = "progress")
     private int progress;
 
+    /**
+     * flag indicating if a student started the learning path
+     */
+    @Column(name = "started_by_student")
+    private boolean startedByStudent = false;
```

Comment:
Should we move this to the future LearnerProfile, that is stored once per student? Or should they get the explanation in each course?

Reply:
Makes sense for the future. @N0W0RK can you add that point to your backlog?